packages/form-validations/README.md

```diff
@@ -0,0 +1,100 @@
+# @lmc-eu/spirit-form-validations
+
+> Form validations for Web and Web-Twig packages of the Spirit Design System.
+
+## Install
+
+```shell
+yarn add @lmc-eu/spirit-form-validations
+```
+
+or
+
+```shell
+npm install --save @lmc-eu/spirit-form-validations
+```
+
+## Usage
+
+Include the javascript file in your html head or just before the closing body tag
+
+```html
+<script src="dist/bundles/umd/spirit-form-validations.min.js" type="text/javascript"></script>
+```
+
+Or
+
+```html
+<script src="dist/bundles/esm/spirit-form-validations.js" type="module"></script>
+```
+
+Now create some form and validate
+
+```
+window.onload = () => {
+    const form = document.getElementById("form1");
+
+    // create the FormValidations instance
+    const formValidations = new FormValidations(form);
+
+    form.addEventListener('submit', function (event) {
+       event.preventDefault();
+
+       // check if the form is valid
+       const valid = formValidations.validate(); // returns true or false
+    });
+};
+```
+
+Profit
+
+It automatically validates `required`, `min`, `max`, `minlength`, `maxlength` attributes and the value of type attributes like `email`, `number` and more...
+
+FormValidations takes 3 parameters:
+
+- **form** - The form element
+
+- **config** - An object containing the configuration. Default is bootstrap’s configuration which is
+
+<a name="default-config"></a>
+
+```js
+const defaultConfig = {
+  formFieldSelector: '[data-spirit-validate]',
+  errorClass: 'has-error',
+  successClass: 'has-success',
+  errorTextParentSelector: '[data-spirit-validate]',
+  errorTextTag: 'div',
+  errorTextClass: '',
+  dataElementMessage: 'validation-text',
+};
+```
+
+- **live** - A boolean value indicating whether FormValidations should validate as you type, default is `true`
+
+## Built-in validators
+
+| Name      | Usage Description                                                                                           |
```

Comment:
I would split this table into three columns: name, usage and description. Same as it's in the [Pristine docs](https://pristine.js.org/#:~:text=%2C%20max%20etc.-,Built%2Din%20validators,-Name).

Reply:
Fixed.